### PR TITLE
Docs (Tour Update): Add a GraphQL tutorial to the Dgraph Tour using a Dgraph Cloud backend

### PR DESCRIPTION
**Pull request by ghost**
Adds a GraphQL tutorial running a Dgraph Cloud backend, along with various enhancements.

Staging link: https://deploy-preview-351--dgraph-tour.netlify.app/tour/

Still to-do:
- [ ] tracking
- [x] widen content pane on tutorial pages, 
- [x] investigate whether we can use a top-and-bottom arrangement for query and results panes (like the Go tutorial)
- [ ] SEO

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/tutorial/351)

 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 

Tour Preview: [Dgraph Preview](https://dgraph-tour-3ea46ecbb5-129710.surge.sh)

### CODE REVIEW

**Comment by bucanero** [comment]
Staging link: https://deploy-preview-351--dgraph-tour.netlify.app/tour/

File: content/graphqlintro/6.md [hunk 1]
```diff
@@ -17,12 +17,7 @@ Graphs describe objects and the interconnections between them. You may have
 heard of friendship graphs, or a social network graph, where data points
 (people) are connected together through known relationships.
 
-Press **Run** on the query to the right. In the resulting graph, circles denote
-nodes (people and animals) and lines denote connections or labeled relationships
-between these people and animals.
-
-Graphs lend themselves so naturally to visualizations that you'll find yourself
-thinking in terms of graphs and visualizations of graphs.
+Press **Run** on the query to the right.
```
Comment (ghost) [inline content/graphqlintro/6.md]: I was going to try to get a graph representation of the json like is in Ratel, but believe that is outside of the scope here for right now.

File: content/graphqlintro/1.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+using Dgraph Cloud, a Dgraph database backend that we offer as a managed
+service.
+
+{{% notice "note" %}} To take a tour of Dgraph using Dgraph Query Language (DQL,
+formerly known as GraphQL+-), please see the [DQL-focused portion of this
+tour]({{< relref "intro/1" >}}). The first five sections of this tour focus
+exclusively on using Dgraph Cloud with GraphQL. {{% /notice %}}
+
+The tutorial is divided into modules. You can access the table of contents at
+any time by clicking "A Tour of Dgraph" in the top left and can move backwards
```
Comment (MichelDiz) [inline content/graphqlintro/1.md]: This is confusing. I know it is from the previous text. But we should change it now. The UI doesn't reflect this phrase.

File: content/graphqlmoredata/3.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
++++
+date = "2020-09-15T00:00:00+00:00"
+next = "/?graphql"
+prev = "/graphqlmoredata/3"
```
Comment (MichelDiz) [inline content/graphqlmoredata/3.md]: This step is not showing up. By clicking we are redirected before the page appears.

**Review by MichelDiz — CHANGES_REQUESTED** [review]

File: content/graphqlintro/1.md (hunk 2 above)
Comment (ghost) [reply content/graphqlintro/1.md]: Done. removed tour navigation instructions as they are not needed anyways IMO.

File: content/graphqlmoredata/3.md (hunk 3 above)
Comment (ghost) [reply content/graphqlmoredata/3.md]: Fixed. This 3rd lesson was added after moving the module. The 2nd lesson now links to the 3rd and the 3rd links back to the 2nd. https://deploy-preview-351--dgraph-tour.netlify.app/tour/graphqlmoredata/3/

**Comment by MichelDiz** [comment]
@verneleem please sign the CLA

**Review by MichelDiz — APPROVED** [review]

File: content/basic/10.md [hunk 4]
```diff
@@ -4,7 +4,10 @@ next = "/basic/11"
 prev = "/basic/9"
 title = "Count"
 weight = 10
+course = 2
+courseName = "Basic"
```
Comment (bucanero) [inline content/basic/10.md]:
```suggestion

```
